resolwe_bio/processes/import_data/sra_file.py

```diff
@@ -194,6 +208,15 @@ def run(self, inputs, outputs):
         """Run the analysis."""
         fastq_gz = []
         for srr in inputs.sra_accession:
+            if inputs.advanced.prefetch:
+                # prefetch the SRR bundle
+                return_code, _, _ = Cmd["prefetch"][
+                    "-p", "--max-size", inputs.advanced.max_size_prefetch, srr
```

Comment:
Maybe list each option-value in a new row, if black permits?

Reply:
It is auto-formatted by Black. One option per row is not possible.